.github/workflows/cd.yaml

```diff
@@ -0,0 +1,30 @@
+# .github/workflows/cd.yml
```

Comment:
Is this comment necessary?
```suggestion
```

Reply:
no~~, thanks!!

Reply:
fixed: https://github.com/route06/aws-cost-cli/pull/13/commits/a8e9029d61489325ef1c19a2d016d8c0537ab47b